app/controllers/notification_subscriptions_controller.rb

```diff
@@ -5,6 +5,7 @@ def create
     @group.subscriptions = params[:notification_subscription_group][:subscriptions]
     @group.subscription_types = params[:notification_subscription_group][:subscription_types]
     @group.user = current_user
+    @group.journey = params[:notification_subscription_group][:journey_id]
```

Comment:
Toto je nebezpecne, urcite to treba najst v DB a az potom tam asociovat. Inak moze niekto podstrcit lubovolne id.